client/commonFramework/output-display.js

```diff
@@ -37,14 +50,16 @@ window.common = (function(global) {
 
   codeOutput.setSize('100%', '100%');
 
-  common.updateOutputDisplay = function updateOutputDisplay(str = '') {
-    codeOutput.setValue(str);
-    return str;
+  common.updateOutputDisplay = function updateOutputDisplay(displayVal = '') {
+    const convertedDisplayVal = convertDisplayValue(displayVal);
+    codeOutput.setValue(convertedDisplayVal);
+    return convertedDisplayVal;
   };
 
-  common.appendToOutputDisplay = function appendToOutputDisplay(str = '') {
-    codeOutput.setValue(codeOutput.getValue() + str);
-    return str;
+  common.appendToOutputDisplay = function appendToOutputDisplay(displayVal = '') {
```

Comment:
Not always but it is best practice. 

@yjcxy12 Please rebase this PR and fix all lint errors